mlflow/ml-package-versions.yml

```diff
@@ -328,3 +328,14 @@ spark:
         ./dev/setup-spark-datasource-autologging.sh
         find tests/spark_autologging/datasource -name 'test*.py' | xargs -L 1 pytest --large
       fi
+
+mleap:
+  package_info:
+    pip_release: "mleap"
+
+  models:
+    minimum: "0.15.0"
+    maximum: "0.17.0"
+    requirements: ["pyspark==2.4.5"]
+    run: |
+      pytest tests/spark/test_mleap_model_export.py --large
```

Comment:
@dbczumar 

`tests/spark/test_mleap_model_export.py::test_model_deployment` requires building `mlflow-pyfunc:latest` image first:

```
Unable to find image 'mlflow-pyfunc:latest' locally
```

https://github.com/mlflow/mlflow/pull/4721/checks?check_run_id=3395138870#step:11:193

Reply:
I think we can move `tests/spark/test_mleap_model_export.py::test_model_deployment` in `tests/sagemaker`.